STATUS: INLINE SUCCESSFUL
MODIFIED: com.github.kevinsawicki.http.HttpRequestTest.basicProxyAuthentication
MODIFIED FILE: /home/anonymous/projects/tools/reseter-finder/exp/_downloads/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
CLEANER: com.github.kevinsawicki.http.TestGroup100Case0.test1
CLEANER FILE: /home/anonymous/projects/tools/reseter-finder/exp/_downloads/http-request/lib/src/test/java/com/github/kevinsawicki/http/TestGroup100Case0.java
POLLUTER: com.github.kevinsawicki.http.HttpRequestTest.customConnectionFactory
POLLUTER FILE: /home/anonymous/projects/tools/reseter-finder/exp/_downloads/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 2
ELAPSED TIME: 68.80100011825562
==========================
@@ -966,0 +966,4 @@
+ {
+     com.github.kevinsawicki.http.HttpRequest.ConnectionFactory connectionFactory0 = null;
+     com.github.kevinsawicki.http.HttpRequest.setConnectionFactory(connectionFactory0);
+ }
